### PR TITLE
[Ubuntu] always install gfrotran

### PR DESCRIPTION
**Pull request by mikhailkoliada**
# Description

Always install latest Fortran symlink

## Check list
- [X] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/ubuntu/scripts/build/install-gfortran.sh
Comment (Chalakadir) [inline images/ubuntu/scripts/build/install-gfortran.sh:1]: Best wishes for everything

**Review by Chalakadir — APPROVED** [review]

**Review by Chalakadir — APPROVED** [review]

**Review by shamil-mubarakshin — APPROVED** [review]

**Review by Chalakadir — COMMENTED** [review]
### ****

File: images/ubuntu/scripts/build/install-gfortran.sh
Comment (Chalakadir) [inline images/ubuntu/scripts/build/install-gfortran.sh:1]: [Skip to content](https://github.com/actions/runner-images/pull/10069/files#start-of-content)
Navigation Menu

Code
Issues
39

You need to leave a comment indicating the requested changes. 
Jump to bottom
[Ubuntu] always install gfrotran #10069
 Merged
[shamil-mubarakshin](https://github.com/shamil-mubarakshin) merged 1 commit into [main](https://github.com/actions/runner-images/tree/main) from [ubuntu24_gfortran](https://github.com/actions/runner-images/tree/ubuntu24_gfortran)  2 hours ago
[Conversation 7](https://github.com/actions/runner-images/pull/10069)
[Commits 1](https://github.com/actions/runner-images/pull/10069/commits)
[Checks 10](https://github.com/actions/runner-images/pull/10069/checks)
[Files changed 1](https://github.com/actions/runner-images/pull/10069/files)
File filter 
 
  3 changes: 3 additions & 0 deletions3  
images/ubuntu/scripts/build/install-gfortran.sh
Original file line number	Diff line number	Diff line change
@@ -14,4 +14,7 @@ for version in ${versions[*]}; do
    apt-get install $version
done

echo "Install versionless gfortran (latest)"
apt-get install gfortran

invoke_tests "Tools" "gfortran"
Footer
© 2024 GitHub, Inc.
Footer navigation
[Terms](https://docs.github.com/site-policy/github-terms/github-terms-of-service)
[Privacy](https://docs.github.com/site-policy/privacy-policies/github-privacy-statement)
[Security](https://github.com/security)
[Status](https://www.githubstatus.com/)
[Docs](https://docs.github.com/)
[Contact](https://support.github.com/?tags=dotcom-footer)
Manage cookies
Do not share my personal information

Comment (Chalakadir) [reply images/ubuntu/scripts/build/install-gfortran.sh:1]: Good work keep on

**Review by Chalakadir — COMMENTED** [review]
### ****

**Comment by Chalakadir** [comment]
> ### **__**

**Review by Chalakadir — COMMENTED** [review]
1. > ` 0938621521`

**Comment by Chalakadir** [comment]
I 'm very interesting use github

File: images/ubuntu/scripts/build/install-gfortran.sh
Comment (Chalakadir) [inline images/ubuntu/scripts/build/install-gfortran.sh:1]: Image/Ubuntu/scripts/build/install

**Pull request merged**